ext/intl/converter/converter.c

```diff
@@ -124,10 +126,12 @@ PHP_METHOD(UConverter, fromUCallback) {
 	zval *source, *error;
 	zend_long codePoint;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "lalz",
-		&reason, &source, &codePoint, &error) == FAILURE) {
-		RETURN_THROWS();
-	}
+	ZEND_PARSE_PARAMETERS_START(4, 4)
+		Z_PARAM_LONG(reason)
+		Z_PARAM_ARRAY(source)
```

Comment:
Unrelated, but am I blind or is the `source` parameter never used?

Reply:
yes when I did the changes I saw it too I had a little pause when I realised :) esp. considering below it is not a macro.

Reply:
Peak :|